scss/core/tokens/_sizing.scss

```diff
@@ -0,0 +1,54 @@
+// stylelint-disable declaration-no-important
+
+// Width and height
+
+@each $prop, $abbrev in (width: w, height: h) {
+  @each $size, $length in $sizes {
+    .#{$abbrev}-#{$size} { #{$prop}: $length !important; }
+  }
+}
+
+//Widths
+
+.mw-100 { max-width: 100% !important; }
+.mh-100 { max-height: 100% !important; }
+
+.w-auto{
+  width: auto;
+}
+
+@each $size, $value in $widths {
+  .w-#{$size} {
+    width: 1rem * $value !important;
+  }
+}
+
+// Maya icon sizes
+
+.dci {
+  vertical-align: middle;
+  font-size: 1em;
+}
+
+@each $size, $value in $icon-size {
+  .dci-#{$size} {
+    font-size: 1em * $value !important;
+  }
+}
+
+.de-code{
+  @each $size, $value in $widths {
+    .w-#{$size} {
+      width: 1rem * $value !important;
+      background-color: $gray-100;
+      height: 25px;
+      margin-bottom: 5px;
+    }
+  }
+  .w-auto{
+    background-color: $gray-100;
+    height: 25px;
```

Comment:
w-auto comes with bootstrap units, we may not need to add it again here. Need to check.
https://getbootstrap.com/docs/4.4/utilities/sizing/#relative-to-the-parent

Reply:
i found the same class in line number 16.

Reply:
I checked. `w-auto` CSS is not defined. It might be in the new versions of bootstrap